utils_test.go

```diff
@@ -0,0 +1,60 @@
+package orientation
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/sirupsen/logrus"
+)
+
+func TestF3(t *testing.T) {
+	F3Img, err := getImageFromFilePath("./TestImages/F-3.jpg")
+	if err != nil {
+		fmt.Printf("error: %s", err)
+	}
+	imgToSaveBytes, orientation := ReadImage(F3Img, logrus.NewEntry(logrus.StandardLogger()))
+
+	os.WriteFile("./TestImages/f3_after_tests.jpg", imgToSaveBytes, os.ModeDevice.Perm())
+	if orientation == "none" {
+		t.Logf("test F3 passed with orientation %s", orientation)
+		return
+	}
+	t.Errorf("Expected no orientation, found: %s", orientation)
+
+}
+
+func TestF1(t *testing.T) {
+	F1Img, err := getImageFromFilePath("./TestImages/F-1.jpg")
+	if err != nil {
+		fmt.Printf("error: %s", err)
+	}
+	_, orientation := ReadImage(F1Img, logrus.NewEntry(logrus.StandardLogger()))
+
+	if orientation == "1" {
+		return
+	}
+	t.Errorf("Expected orientation 1, found: %s", orientation)
+}
+
+func TestFnone(t *testing.T) {
```

Comment:
This test is the same as: `TestF3` ? both make the same `assert IF =D`

Reply:
Not exactly, the output is the same, but this one should return earlier and uses less processing if the image already has no orientation, where as TestF3 assumes an image input with `orientation 3` and does the required steps to set it to none

I should've made a better assert but for now I'm just trying to get this to work tbh